tests/x11/gnomecase/application_starts_on_login.pm

```diff
@@ -181,22 +181,24 @@ sub run {
     handle_relogin;
     assert_screen "generic-desktop";
 
-    # save session
-    $self->alter_status_auto_save_session;
-    x11_start_program('xterm');
-    assert_screen 'xterm';
-    handle_relogin;
-    wait_still_screen;
-    send_key 'alt-tab';    # select xterm if unselected
-    assert_screen 'xterm';
-    send_key 'alt-f4';
-    wait_still_screen 2, 4;
-
-    if (is_sle('12-SP2+')) {
-        $self->restore_status_auto_save_session;
-    }
-    else {
+    # save session, available only for GNOME<3.34, see bsc#1158851
```

Comment:
Based on the input from developer "The upstream disables the session save/restore function on gnome-session 3.34.2"
So, please make the comment more precise.